sandbox/user_guide.md

```diff
@@ -14,26 +14,24 @@ sections:
-
-## Getting Started with APIs
-
-* What’s an API?
-  * An API (Application Programming Interface) is a set of features and rules that exist inside a software program that allows other software programs to interact with it. For example, you can build an app that uses the Twitter API to get information or data from a user's Twitter account. APIs are used in a wide variety of ways, but for our purposes, you can think of an API as a pipeline that can allow your ACO’s computer systems to receive data directly from CMS’ databases.
-  * Need more information about APIs? Here are some great introductions:
-    * [Introduction to Web APIs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction){:target="_blank"}
-    * [An Intro to APIs](https://www.codenewbie.org/blogs/an-intro-to-apis){:target="_blank"}
-* Do I need to know how to code to use BCDA?
-  * You do not need to know how to code!  Our documentation is written so that everyone -- regardless of technical exposure -- can access beneficiary data. For this walkthrough, we’ll be using a platform called Swagger, where you’ll be able to interact with the API through a web interface.
+## Getting started with APIs
+  * What’s an API?
+    * An API (Application Programming Interface) is a set of features and rules that exist inside a software program that allows other software programs to interact with it. For example, you can build an app that uses the Twitter API to get information or data from a user's Twitter account. APIs are used in a wide variety of ways, but for our purposes, you can think of an API as a pipeline that can allow your ACO’s computer systems to receive data directly from CMS’ databases.
+    * Need more information about APIs? Here are some great introductions:
+      * [Introduction to Web APIs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction){:target="_blank"}
+      * [An Intro to APIs](https://www.codenewbie.org/blogs/an-intro-to-apis){:target="_blank"}
+  * Do I need to know how to code to use BCDA?
+    * You do not need to know how to code!  Our documentation is written so that everyone -- regardless of technical exposure -- can access beneficiary data. For this walkthrough, we’ll be using a platform called Swagger, where you’ll be able to interact with the API through a web interface.
 
 ## Setting up your credentials in Swagger
 
 ### Navigating to Swagger
 
-To get to BCDA's Swagger page for synthetic data, click on the following link, or copy and paste it into your web browser:
+To get to BCDA's Swagger page for sandbox data, click on the following link, or copy and paste it into your web browser:
```

Comment:
Why did this change?

Reply:
I'm honestly not sure! I have it changed back.